src/functions-templates/go/scheduled-function/Makefile

```diff
@@ -0,0 +1,7 @@
+build:
+	mkdir -p functions
+	GOOS=linux
+	GOARCH=amd64
+	GO111MODULE=on
+	GOBIN=${PWD}/functions go get ./...
+	GOBIN=${PWD}/functions go install ./...
```

Comment:
is this needed? i thought the CLI was able to build go functions on its own now?

Reply:
Correct, it should be fine to remove.